VimeoNetworking/Sources/ErrorCode.swift

```diff
@@ -68,6 +68,19 @@ public enum VimeoErrorCode: Int
     case facebookInvalidToken = 2300
     case facebookMissingProperty = 2301
     case facebookMalformedToken = 2302
+    case googleUnableToCreateUserMissingEmail = 2325
```

Comment:
Can 3rd party apps use Google auth too?

Reply:
No but good point. I tried to isolate the requests that send the token to our backend, those are in our app. But didn't have a way to extend this enum without doing it here.